Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -31,8 +35,17 @@ web page context.
 
     *   https://www.youtube.com/watch?v=a_cS2XfCplI&t=28766s
 
+CSP rules are used to describe, which external links or functionality are allowed for
+certain HTML tags (like :html:`<script>`, :html:`<img>`, :html:`<iframe>`). This allows
+to restrict external resources or JavaScript execution with security in mind. When
+accessing a page, these rules are sent as part of the HTTP Request from the server to
+the browser, and the browser will enforce these rules (and reject non-allowed content).
+This rejection can also be logged.
+
 Content Security Policy declarations can be applied to a TYPO3 website in
-frontend and backend scope with a dedicated API.
+frontend and backend scope with a dedicated API. This API allows for specific, site or
+extension-specific configuration instead of manually setting the CSP rules with
+server-side configuration through `httpd.conf/nginx.conf` or `.htaccess` files.
 
 To delegate Content Security Policy handling to TYPO3 frontend, at least one of
```

Comment:
This is not true anymore. SInce v12.4.20, you can enable site-specific configuration. like here:
https://brotkrueml.dev/presentations/t3cmd25/csp_typo3.html#/24

Then you don't have to enable the feature settings.